src/main/java/ru/itis/security/util/JwtTokenUtil.java

```diff
@@ -0,0 +1,61 @@
+package ru.itis.security.util;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Component
+public class JwtTokenUtil implements Serializable {
```

Comment:
I see no difference between these classes. Enterpriseee.
JwtTokenUtil, JwtAuthenticationEntryPoint, JwtAuthenticationProvider, JwtRequestFilter, JwtAuthenticationFilter. One responsibility smudged into 5 classes.

Reply:
The single class will be overloaded, I suppose

Reply:
removed JwtAuthenticationEntryPoint